src/flows/components/FromTemplatePage.tsx

```diff
@@ -33,6 +33,263 @@ const TEMPLATE_MAP = {
+              },
+              buckets: [],
+              functions: [
+                {
+                  name: 'mean'
+                }
+              ],
+              tags: [
+                {
+                  aggregateFunctionType: 'filter',
+                  key: '_measurement',
+                  values: []
+                }
+              ]
+            },
+            editMode: 'builder',
+            name: '',
+            text: ''
+          }
+        ],
+        shape: 'chronograf-v2',
+        showNoteWhenEmpty: false,
+        staticLegend: {
+          heightRatio: 0.2,
+          hide: true,
+          widthRatio: 1
+        },
+        type: 'xy',
+        xColumn: null,
+        xTickStart: null,
+        xTickStep: null,
+        xTotalTicks: null,
+        yColumn: null,
+        yTickStart: null,
+        yTickStep: null,
+        yTotalTicks: null
+      },
+      type: 'visualization',
+      title: 'Visualize response_time',
+      visible: true
+    },
+    {
+      mode: 'preview',
+      text: 'One of the most common use cases for InfluxDB is downsampling data to reduce the overall disk usage as data collects over time.\n\nThis is done by aggregating data within windows of time, then storing the aggregate value in a new bucket.',
+      type: 'markdown',
+      title: 'Step 2 - Let\'s Downsample',
+      visible: true
+    },
+    {
+      activeQuery: 0,
+      queries: [
+        {
+          builderConfig: {
+            buckets: [],
+            functions: [],
+            tags: []
+          },
+          editMode: 'advanced',
+          text: '// __Previous_Result__ uses data from all the panels above it as an input to the Flux function\n__PREVIOUS_RESULT__\n\n  // This Flux function windows and aggregates the data in to 15m averages\n|> aggregateWindow(fn: mean, every: 15m)'
+        }
+      ],
+      type: 'rawFluxEditor',
+      title: 'Downsample with Flux',
+      visible: true
+    },
+    {
+      mode: 'preview',
+      text: 'You might want to keep your downsampled data separate from your raw data with a longer retention policy.\n\nSelect a separate bucket or create a new one from the dropdown in the next output cell.\n\nThis generates:\n\n`|> to(bucket: \'YOUR NEW BUCKET\')`\n\nBonus: You can switch the notebook mode from Preview to Run to write this data to the new bucket manually.\n',
+      type: 'markdown',
+      title: 'Step 3 - Select Output',
+      visible: true
+    },
+    {
+      bucket: {
+        createdAt: '2021-05-24T22:00:11.407744829Z',
+        id: '031437230cc0b68b',
+        labels: [],
+        links: {
+          labels: '/api/v2/buckets/031437230cc0b68b/labels',
+          members: '/api/v2/buckets/031437230cc0b68b/members',
+          org: '/api/v2/orgs/e859306b862b3e21',
+          owners: '/api/v2/buckets/031437230cc0b68b/owners',
+          self: '/api/v2/buckets/031437230cc0b68b',
+          write: '/api/v2/write?org=e859306b862b3e21&bucket=031437230cc0b68b'
```

Comment:
Not sure how these work with a template... right now these all map to the org that I used to create and export the JSON...
`Member`
`Org`
`Owners`
`Self`
`Write`
?

Reply:
Ok what if we use the `_tasks` bucket? I think it's a system bucket that everyone has

Reply:
none of these links are needed. feel free to delete the whole object if needed

Reply:
Done